admin/card/feature/class-timely-auto-updates.php

```diff
@@ -37,7 +37,7 @@ public function init() {
 		// Determine whether or not this feature is enabled.
 		$auto_update_settings = $core->settings->get_setting( 'auto_update' );
 
-		$is_enabled = isset( $auto_update_settings['timely-updates-enabled'] ) && '1' === $auto_update_settings['timely-updates-enabled'];
+		$is_enabled = ! empty( $auto_update_settings['timely-updates-enabled'] ) && '1' === $auto_update_settings['timely-updates-enabled'];
```

Comment:
```
! empty( $auto_update_settings['timely-updates-enabled'] ) &&
'1' === $auto_update_settings['timely-updates-enabled'];
```

You shouldn't need both checks. The `empty()` should catch if the `timely-updates-enabled` key doesn't exist or it equals `0` or `"0"`.

How about just:
```
$is_enabled = ! empty( $auto_update_settings['timely-updates-enabled'] );
```

Reply:
mkay. i dunno why I was thinking that 0 was empty but "0" was not. got it